hygiene-safety-guidelines.md

```diff
@@ -0,0 +1,47 @@
+---
+layout: default
+title: co.up hygiene and safety guidelines
+---
+
+<h1>
+  Hygiene & safety guidelines
+ </h1>
+ 
+*updated on July 22, 2021* 
+
+In light of the current public health situation, the following regulations are in place at co.up:
+
+**Maximum occupancy**<br/>
+Maximum occupancy at co.up is currently 12 people for all in-person events. This includes guests, speakers and organizers.
+We ask that organizers make sure to limit signups accordingly. In order to add to the accessibility of their events, we also encourage them to offer them in a hybrid format (IRL + streaming) when possible.
+
+**Hygiene concept and enforcement**<br/>
+Organizers must prepare a personalized hygiene concept for their event and are solely responsible for posting it, enforcing it, and submitting it to local health authorities upon request. This concept must follow the guidelines listed in Berlin's [Dritte SARS-CoV-2-Infektionsschutzmaßnahmenverordnung](https://www.berlin.de/corona/massnahmen/verordnung/), §5. Organizers can use our guidelines text as a basis for their hygiene concept; their own concept must be aligned with co.up's.
+
+**Contact tracing and documentation**<br/>
+Each person present in the space during an event must provide the following information: first and last name, phone number, district or city of residence, e-mail address (if available), full residential address (if available), times of attendance and their seat or table number (if applicable). Organizers are required to collect this information, to store it securely for four weeks, and to provide it to local health authorities upon request.
+
+**Proof of vaccination, recovery, or negative test result**<br/>
+As of July 6, 2021, only guests at events of 50 people and up are legally required to provide certification of either a negative test result, full vaccination, or recovery from covid-19. This is therefore currently not mandatory for events at co.up. We do, however, encourage attendees and organizers to make use of Berlin's free antigen testing offer on the day of their event. Guests presenting any cold-like symptoms, and/or an impacted sense of taste or smell, on the day of the event are not allowed to enter the space.
+
+**Masks**<br/>
+FFP2 masks are mandatory for everyone in the space at all times, including when you are seated and with the exception of performing or presenting. This mandate applies to the entirety of the event space, the meeting room, the kitchen and the restrooms. There are FFP2 masks available to use in the space. Our landlord also mandates wearing a medicinal mask on the entirety of the building premises (i.e. the stairwell, the elevator, and the courtyards and parking lots).
```

Comment:
we do not provide masks for events, that is responsibility of organizers (at max a emergency supply limited to 3). 

the meeting room is closed for events so it should not be mentioned here.